pkg/cmd/flags/output.go

```diff
@@ -30,92 +34,82 @@ option:{stack-addresses,exec-env,relative-time,exec-hash,parse-arguments,sort-ev
   parse-arguments                                  do not show raw machine-readable values for event arguments, instead parse into human readable strings
   parse-arguments-fds                              enable parse-arguments and enrich fd with its file path translation. This can cause pipeline slowdowns.
   sort-events                                      enable sorting events before passing to them output. This will decrease the overall program efficiency.
+
 Examples:
   --output json                                            | output as json to stdout
-  --output gotemplate=/path/to/my.tmpl                     | output as the provided go template
-  --output out-file:/my/out --output log-file:/my/log      | output to /my/out and logs to /my/log
+  --output json:/my/out                                    | output as json to /my/out
+  --output gotemplate=/path/to/my.tmpl                     | output as the provided go template to stdout
+  --output gob:/my/out --output log-file:/my/log      	   | output gob to /my/out and logs to /my/log
+  --output json --output gob:/my/out                       | output as json to /my/out
+  --output json:/my/out1,/my/out2                          | output as json to both /my/out and /my/out2
```

Comment:
Tested as:

`--output json:./out1,./out2`
`--output json:stdout,./output`
`--output json:./output --output json`

And one of them doesn't outputs; it's outputting to one end only.

Reply:
@geyslan I've tested the samples above and they all worked. 🤔